entrypoint.sh

```diff
@@ -23,7 +23,12 @@ cloud-api-adaptor-aws aws \
 	-aws-secret-key ${AWS_SECRET_ACCESS_KEY} \
 	-aws-region ${AWS_REGION} \
 	-pods-dir /run/peerpod/pods \
-	-aws-lt-name ${PODVM_LAUNCHTEMPLATE_NAME:-kata} \
+	-use-lt false \
+	-imageid ${PODVM_AMI_ID} \
```

Comment:
I wonder whether we should provide some prebuilt public image at some point..
I suppose the instructions we already have to crate the image are still valid, however i just recalled they are missing the requirement to install aws-cli maybe it would be worth to add it as part of this series

Reply:
> provide some prebuilt public image at some point..
Do you mean AMI ?

Reply:
yes

Reply:
`aws-cli` is not mandatory. One can create the required AWS infra (VPC, SG etc) using the UI as well. Packer doesn't uses `aws-cli`.